include/qpdf/QPDFObjectHandle.hh

```diff
@@ -1629,15 +1641,9 @@ QPDFObjectHandle::isIndirect() const
 }
 
 inline bool
-QPDFObjectHandle::isInitialized() const noexcept
+QPDFObjectHandle::isInitialized() const
```

Comment:
For my edification, why remove `noexcept`?

Reply:
I found that there is a noticeable and surprising performance cost to noexcept .